config.sample.yaml

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Weren't the older .env files were copied somehow when the devcontainer started?

Reply:
Fixed